src/app/public/modules/datepicker/datepicker-input.directive.ts

```diff
@@ -161,6 +161,20 @@ export class SkyDatepickerInputDirective
     return this._startingDay || this.configService.startingDay;
   }
 
+  /**
+   * Requires date value to strictly match the provided dateFormat. When set to false,
+   * if the datepicker directive cannot find an exact match, it will attempt to format the string using the ISO8601 standard format.
+   * @default false
```

Comment:
☝️ @blackbaud-johnly or @Blackbaud-DeniseCisneros curious on your thoughts for this property description. It's not great, but I had to start somewhere.

Reply:
I'd tweak the first sentence to follow our standard verbiage for booleans: "Indicates whether to require..." Also, we're missing "the" before "date value." And do we need "strictly"? Also, shouldn't "provided" be "specified"? I mean, "provided" sounds to me like end users are providing a format and not that the `dateFormat` property is being set for the directive. But maybe I'm missing something here. Also, I'm pretty sure we should either format "dateFormat" as a code snippet and add "value" after it or change it to plain text "date format."

In the second sentence, I'm pretty sure "false" should be a code snippet and that "datepicker directive" should be "datepicker input directive." Also, the sentence structure feels a bit wonky to me. What do you think about tweaking it to something like "If this property is `false` and the datepicker input directive cannot find an exact match, it attempts to format the string based on the ISO 8601 standard format." (Note that I added between a space between "ISO" and "8601" because that seems to be the standard format in search results.)

A few questions:
- We say what happens in the "false" scenario. What happens in the "true" scenario? I assume the "true" scenario means that if the dateFormat is not followed strictly, an error is thrown for the field. Should we make this explicit in the docs?
- In the second sentence, what do we mean when we say that the directive "cannot find an exact match." I left that wording in place, but I don't really understand what we mean or when this happens. The issue that prompted this update makes it seem like this property is related to validation, but we don't say that explicitly here. I may be misunderstanding what's going on here, but I wonder if this second sentence should be changed to something more like "If this property is `false`, then the datepicker input directive attempts to format any date values that do not match the date format specified by the `dateFormat` property using the the ISO 8601 standard format." Or something like that. But again, I may not be following what's happening here.
- Should we link to info on the ISO standard? Something like https://www.iso.org/iso-8601-date-and-time-format.html? Also, should "ISO8601" have

Reply:
Oh, and datepicker is still in the phase 1 version of the updated docs, so after we settle on the wording here, Denise or I can set up a corresponding PR in the doc repo to make the change there as well.